rest-new/src/main/java/org/jboss/pnc/rest/endpoints/internal/DebugEndpointImpl.java

```diff
@@ -75,4 +78,23 @@ public static Build newBuild() {
                 .temporaryBuild(true)
                 .build();
     }
+
+    @Override
+    public void throwEx() throws Exception {
+        RuntimeException nested = new RuntimeException("Root exception.");
+        throw new Exception("Test exception.", nested);
+    }
+
+    @Override
+    public void nocontent() throws Exception {
+        Response.status(Response.Status.NO_CONTENT).build();
```

Comment:
you build a response here but then do nothing with it. 
From JAX-RS specification
> 3.3.3 Return Type
> `void` Results in an empty entity body with a 204 status code.

So Having just empty method will do the works.

Reply:
thanks!